src/util.cc

```diff
@@ -133,17 +135,30 @@ void LowMemoryNotification() {
   }
 }
 
-std::string GetHumanReadableProcessName() {
-  char name[1024];
-  GetHumanReadableProcessName(&name);
-  return name;
+std::string GetProcessTitle(const char* default_title) {
+  std::string buf(16, '\0');
+
+  for (;;) {
+    const int rc = uv_get_process_title(&buf[0], buf.size());
```

Comment:
Not blocking for this PR, but... in the future, might it be possible to introduce a variant of `uv_get_process_title` that can optionally report the size of the buffer required? something along the lines of...

```c
  size_t actual = buf.size();
  const int rc = uv_get_process_title2(&buf[0], &actual);
  if (rc == UV_ENOBUFS) {
    buf.resize(actual);
    // then try again
  }
  // ....
```

Definitely minor but a bit nicer to avoid the multiple resize operations